templates/frameworks/list.html.twig

```diff
@@ -113,98 +111,45 @@
-                        {#10#}
-                        {#</label>#}
-                        {#</div>#}
-                        {#<div class="govuk-radios__item govuk-radios__item--small">#}
-                        {#<input class="govuk-radios__input govuk-radios__input--small" id="results-per-page-2" name="where-do-you-live" type="radio" value="scotland">#}
-                        {#<label class="govuk-label govuk-radios__label" for="results-per-page-2">#}
-                        {#25#}
-                        {#</label>#}
-                        {#</div>#}
-                        {#<div class="govuk-radios__item govuk-radios__item--small">#}
-                        {#<input class="govuk-radios__input govuk-radios__input--small" id="results-per-page-3" name="where-do-you-live" type="radio" value="wales">#}
-                        {#<label class="govuk-label govuk-radios__label" for="results-per-page-3">#}
-                        {#50#}
-                        {#</label>#}
-                        {#</div>#}
-                        {#<div class="govuk-radios__item govuk-radios__item--small">#}
-                        {#<input class="govuk-radios__input govuk-radios__input--small" id="results-per-page-4" name="where-do-you-live" type="radio" value="northern-ireland">#}
-                        {#<label class="govuk-label govuk-radios__label" for="results-per-page-4">#}
-                        {#View all#}
-                        {#</label>#}
-                        {#</div>#}
-                        {#</div>#}
-                        {#</fieldset>#}
-                        {#</div>#}
-
-
                     </form>
 
                 </div>
-                <div class="govuk-grid-column-two-thirds">
-
-                    {% if pagination.totalResults == 1 %}
-                        <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }}
-                            framework found</h2>
+                {% if guided_match_flag is defined %}
+                    {% if query is defined and query in keywords%}
+                    <div class="govuk-grid-column-two-thirds guided-match-background govuk-!-margin-bottom-2 guided-match-area">
+                        {% if pagination.totalResults == 1 %}
+                            <h2 class="govuk-heading-m ccs-font-weight-semibold govuk-!-font-size-24">{{ pagination.totalResults }} framework found</h2>
+                        {% else %}
+                            <h2 class="govuk-heading-l ccs-font-weight-semibold text-white govuk-!-margin-1">Get help finding the right framework</h2>
+                            <h2 class="govuk-heading-s ccs-font-weight-semibold text-white">There are "{{ pagination.totalResults }}" results found for {{ query }}. Depending on your requirements, some of these frameworks maybe better suited than others.</h2>
+                        {% endif %}
```

Comment:
This conditional does not seem correct here, only the else path is needed

Reply:
Please read the two different lines again. You asked me to do this since the guided match area shouldn't show up if the query is in key words and only returns one result. This conditional is correct for this path as one is plural and the other singular.